packages/netlify-cms-backend-git-gateway/src/GitHubAPI.ts

```diff
@@ -91,12 +94,20 @@ export default class API extends GithubAPI {
       tree: string;
       parents: string[];
       author?: { name: string; date: string };
+      committer?: { name: string; date: string };
     } = {
       message,
       tree: changeTree.sha,
       parents: changeTree.parentSha ? [changeTree.parentSha] : [],
     };
 
+    if (this.committer) {
+      commitParams.committer = {
```

Comment:
`committer` defaults to `author`, see https://developer.github.com/v3/git/commits/#create-a-commit.

So this seems redundant